tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -2196,6 +2254,11 @@ uint32_t SPIRVEmitter::getValueZero(QualType type) {
     }
   }
 
+  {
```

Comment:
I'm not sure that we should call `getMatElemValueZero` here since the `getValueZero` method returns value zero for the **given** type while `getMatElemValueZero` does not.